pex/pep_425.py

```diff
@@ -52,6 +53,30 @@ class CompatibilityTags(object):
     See: https://www.python.org/dev/peps/pep-0425/#use
     """
 
+    @classmethod
+    def from_wheel(cls, wheel):
```

Comment:
Ah, meant to unit test this. I'll hit that in a follow up unless there is other feedback to address since its all heavily IT'd.

Reply:
Alright, enough good feedback to make this worth adding in too.